app/application.js

```diff
@@ -68,7 +68,7 @@ export class DesktopApplication {
     this.platform = platform;
     this.isMac = Platforms.isMac(this.platform);
     app.name = AppName;
-    app.allowRendererProcessReuse = false;
+    app.allowRendererProcessReuse = true;
```

Comment:
Wait what is this used for? And what are its security implications?

Reply:
It's hard to sum up so here's the thread: https://github.com/electron/electron/issues/18397
Electron used to prevent Chromium default process reuse behavior to work with native modules better. With new ways to write native modules they want to deprecate that functionality. I set the property to `true` to silence the deprecation warning, but really we've never made use of that behavior because we always stay on the same page (so there's no question of process reuse between page changes) and we don't use native modules anymore (so we won't run into compatibility issues)

Reply:
From a security standpoint I think this is a win simply because there will be less custom code on Electron's side to handle navigation.

Reply:
Ah very nice 👍